1-org/envs/shared/log_sinks.tf

```diff
@@ -18,15 +18,16 @@ locals {
   parent_resource_id   = local.parent_folder != "" ? local.parent_folder : local.org_id
   parent_resource_type = local.parent_folder != "" ? "folder" : "organization"
   parent_resources     = { resource = local.parent_resource_id }
-  main_logs_filter     = <<EOF
+  logs_filter          = <<EOF
     logName: /logs/cloudaudit.googleapis.com%2Factivity OR
     logName: /logs/cloudaudit.googleapis.com%2Fsystem_event OR
     logName: /logs/cloudaudit.googleapis.com%2Fdata_access OR
+    logName: /logs/cloudaudit.googleapis.com%2Faccess_transparency OR
+    logName: /logs/cloudaudit.googleapis.com%2Fpolicy OR
     logName: /logs/compute.googleapis.com%2Fvpc_flows OR
     logName: /logs/compute.googleapis.com%2Ffirewall OR
-    logName: /logs/cloudaudit.googleapis.com%2Faccess_transparency
+    logName: /logs/dns.googleapis.com%2Fdns_queries
```

Comment:
as part of this migration why is cloudauit being changed to dns_queries

Reply:
actually the original entry `logName: /logs/cloudaudit.googleapis.com%2Faccess_transparency` was not replaced but moved up so that all `cloudaudit.googleapis.com` stay together in the start of the filter:

```hcl
    logName: /logs/cloudaudit.googleapis.com%2Factivity OR
    logName: /logs/cloudaudit.googleapis.com%2Fsystem_event OR
    logName: /logs/cloudaudit.googleapis.com%2Fdata_access OR
    logName: /logs/cloudaudit.googleapis.com%2Faccess_transparency OR <=here
    logName: /logs/cloudaudit.googleapis.com%2Fpolicy OR
```

the `logName: /logs/dns.googleapis.com%2Fdns_queries` is a new entry to send DNS queries to the sink.